src/strings/en/ui.js

```diff
@@ -3,6 +3,7 @@ export default {
   headline: 'Appeal to the CBDT seeking reassurance regarding Aadhaar-PAN Linking',
   decription: 'Speak For Me: Ask CBDT to withdraw order mandating the linking of PAN and Aadhaar, or, in the alternative, extend the deadline for the linking of the two until such time sufficient safeguards are installed to protect our fundamental right to privacy.',
   explanation: 'Ask CBDT to withdraw order mandating the linking of PAN and Aadhaar, or, in the alternative, extend the deadline for the linking of the two until such time sufficient safeguards are installed to protect our fundamental right to privacy.',
+  email_collection_note: 'Why have we added a BCC to [email address] while sending your email? When you BCC SpeakForMe in your emails to this authority, we add it to our counter, and highlight the number of emails in our media campaigns. We will also keep you posted on any developments that need your attention. You can unsubscribe from these notices if you prefer not to be involved any further. It may become necessary to take this action further with a lawsuit, either at the local High Court level or directly to the Supreme Court. You may then be requested for participation as a litigant, and we will contact you. If the letters sent achieve their purpose, no such action will be needed. In any case, you will receive an acknowledgement mail with an unsubscribe link. We do not store the content of your email, do not share your email address with third parties, and will not be contacting you for any other matter.',
```

Comment:
'SpeakForMe is included as a BCC recipient. We count the number of emails sent, and may contact you if further action is required on your part. The contents of your email are not stored, and your email address will not be shared with anyone else. You may remove us from BCC if you prefer to not be counted or contacted.'

Reply:
Updated in ef7d3e1b8d8f689413658328cccda17fc8ade969